applications/nrf5340_audio/src/modules/Kconfig

```diff
@@ -49,6 +49,53 @@ config I2S_CH_NUM
 
 endmenu # I2S
 
+#----------------------------------------------------------------------------#
+menu "Music Color Synchronization"
+
+config MUSIC_COLOR_SYNC
+	bool "Enabling RGB synchronization with music"
+	default n
+	help
+		"NRFX_PWM0" and "CMSIS_DSP_TRANSFORM" has to be enabled
+		for this functionality to work.
+
+menu "FFT"
+visible if MUSIC_COLOR_SYNC
+
+config FFT_SAMPLE_SIZE
+	int "Sample size for FFT"
+	default 256
+	help
+		 Supported FFT Lengths are 32, 64, 128, 256, 512,
+		 1024, 2048, 4096 and 8192.
+
+config FFT_FLAG_INVERSE_TRANSFORM
+	int "Flag for transform direction"
+	range 0 1
+	default 0
+	help
+		0 for FFT and 1 for IFFT
+
+config FFT_FLAG_BIT_REVERSAL
+	int "Flag for bit reversal of output"
+	range 0 1
+	default 1
+	help
+		0 to disable and 1 to enable bitreversal
+
+config RGB_RED_FREQ_BIN_RANGE
+	int "Freq range for RED RGB LED"
+	range 0 256
+	default 22 
+
+config RGB_GREEN_FREQ_BIN_RANGE
+	int "Freq range for GREEN RGB LED"
+	range 0 256
+	default 100
+
+endmenu # Music sync
```

Comment:
Hmm these endmenu comments does not match?

Reply:
Changed so that they match